docs/Release-Notes-2.16.md

```diff
@@ -0,0 +1,123 @@
+---
+layout: default
+title: Release Notes 2.16
+---
+
+Released on XXX.
+
+## Mock-core-configs 36.4
+
+The biggest change is the removal of `epel-8-*` configs. It has been replaced by several configs: `alma+epel-8-*`, `centos+epel-8-*`, `oraclelinux+epel-8-*`, `rhel+epel-8-*`, `rocky+epel-8-*`. Every config has its pros and cons:
+
+* `alma+epel-8-*` - This uses Alma Linux 8 + Fedora EPEL. It works right off the bat and it is recommended replacement. The only disadvantage is that Koji actually uses RHEL + EPEL for EPEL builds.
+* `centos+epel-8-*` - This uses CentOS 8 + Fedora EPEL. We do **not** recommend to use this config, because [CentOS 8 will reach EOL on December 31, 2021(https://www.centos.org/centos-linux-eol/).
```

Comment:
Minor markup typo.

```suggestion
* `centos+epel-8-*` - This uses CentOS 8 + Fedora EPEL. We do **not** recommend to use this config, because [CentOS 8 will reach EOL on December 31, 2021](https://www.centos.org/centos-linux-eol/).
```

Reply:
Updated.